Trimble.Modus.Components/Controls/TMInput/CustomInput.xaml.cs

```diff
@@ -0,0 +1,551 @@
+using System.Windows.Input;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components.Controls;
+
+public partial class CustomInput : ContentView
+{
+
+    #region Private Properties
+
+    private const double disabledOpacity = 0.4;
+
+    private ValidationResponse _validationResponse;
+    private CustomInput _customInput;
```

Comment:
Can you explain the need of this reference? This just holds`this`.

Reply:
Replaced with this